enterprise/ha/src/main/java/org/neo4j/kernel/ha/DelegateInvocationHandler.java

```diff
@@ -137,7 +138,7 @@ private void log( String format, Object... arguments )
     {
         if ( log.isDebugEnabled() )
         {
-            log.debug( "[" + componentName + "]" + format, arguments );
+            log.debug( "[" + componentName + ", " + hashCode() + "] " + format, arguments );
```

Comment:
At least add a comment about what purpose the `hashCode()` serves in the log message.

And if that purpose is to uniquely identify the instance, perhaps we should be explicit about it and use `System.identityHashCode( this )`.